database/migrations/2016_04_01_133652_create_users_projects_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateUsersProjectsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user_project', function (Blueprint $table) {
```

Comment:
By convention and for laravel magic to work, table name should be 'project_user'